### PR TITLE
Add files

### PR DESCRIPTION
**Pull request by pankominek**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const uploadButton = document.querySelector('#uploadFile');
```
Comment (IwoHerka) [inline js/main.js:1]: 1. This solution does not modify the original file, it only displays modified content in the UI.
2. Original newlines are not preserved, loaded content is display as a one single huge line. This should not be the case.

Comment (pankominek) [reply js/main.js]: Ok. I changed the solution.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+const leftAlign = lines => {
+    let formatedLines = lines.map(line =>
+        line.trim()
+    );
+
+    output(formatedLines);
+}
+
+//simple use
+rightAlign(lines);
```
Comment (IwoHerka) [inline index.js:60]: Cool, but to use it we have to change the code everytime.
How about you take it next level and make configure script via command line arguments?
We would like to have as input path to file and alignment option.

File: index.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+const leftAlign = lines => {
+    let formatedLines = lines.map(line =>
+        line.trim()
+    );
+
+    output(formatedLines);
+}
+
+//simple use
```
Comment (dbisiakowski) [inline index.js:59]: Add space between // and comments.

File: index.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        }
+
+        return newLine.concat(line);
+    });
+
+    output(formatedLines);
+}
+
+const leftAlign = lines => {
+    let formatedLines = lines.map(line =>
```
Comment (dbisiakowski) [inline index.js:52]:
```suggestion
    const formatedLines = lines.map(line =>
```

File: index.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
+const fs = require('fs');
+const data = fs.readFileSync('testContent.txt', 'UTF-8');
```
Comment (dbisiakowski) [inline index.js:2]: I want do it with other file. 
Try to get path to file from arguments.
process.argv[2]

If file doesn't exist user should get an error.

File: index.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+const numberOfSpaces = (longestLine, lineLenght) =>
+    longestLine - lineLenght;
+
+const rightAlign = lines => {
+    let formatedLines = lines.map(line => {
+        let newLine = '';
+        line = line.trim();
+
+        const spaces = numberOfSpaces(longestLine, line.length);
```
Comment (dbisiakowski) [inline index.js:22]: Move const to the top of block.

File: index.js [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+    output(formatedLines);
+}
+
+const centerAlign = lines => {
+    let formatedLines = lines.map(line => {
+        let newLine = '';
+        line = line.trim();
+
+        const spaces = Math.floor(numberOfSpaces(longestLine, line.length) / 2);
```
Comment (dbisiakowski) [inline index.js:39]: Move const to the top of block.

File: index.js [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+const lines = data.split(/\r?\n/);
+
+const longestLine = lines.reduce((prev, next) =>
+    prev.length > next.length ? prev.length : next.length
+);
+
+const output = lines => {
+    const content = lines.join('\r\n');
+    fs.writeFileSync('testContent.txt', content);
+}
```
[the remark this answers is not in the document]
Comment (pankominek) [reply index.js:12]: ok

File: index.js (hunk 3 above)
Comment (pankominek) [reply index.js:59]: I deleted this comment.

File: index.js (hunk 6 above)
Comment (pankominek) [reply index.js]: Ok, changed.

File: index.js [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+        for (let i = 0; i < spaces; i++) {
+            newLine += ' ';
+        }
+
+        return newLine.concat(line);
+    });
+
+    output(formatedLines);
+}
+
```
[the remark this answers is not in the document]
Comment (pankominek) [reply index.js]: Added everywhere.

**Pull request merged**